tensorflow/src/TensorFlow/Session.hs

```diff
@@ -28,7 +28,6 @@ module TensorFlow.Session (
     runSession,
     runSessionWithOptions,
     MonadBuild(..),
-    extend,
```

Comment:
do we need to stop exporting `extend`? this is part of the public API

Reply:
No, I thought it was more correct since using the builder and buffering the new nodes until we add them all at once is an implementation detail now, but I've added it back.